CMSgov/easi-app
# [NOREF] - Custom alert - dismissable

# NOREF

This change is per UX Findings [DOC](https://docs.google.com/document/d/1HSmSXe6Wg7emBg59s3qFn7lINZnComsbpyVBlEJHb3Y/edit#)

## Changes and Description

- Updated custom `<Alert />` to match MINT's with dismissable functionality and default slim props
- Implemented/standardized all instances of `<Alert />`
- Updated unit tests



## How to test this change

- Create success/error messages and verify they are all slim and dismissable (unless few cases where error is explicitly set to `slim=false`

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
